README.md

```diff
@@ -1 +1,7 @@
 # hw01
+## Surrein Maniraju Homework 1
+Hi my name is *Surrein Maniraju*. I am a fourth year Sociology Major in the college. I am taking this class because there are many things I am interested in that require computing skills such as:
```

Comment:
Sharing some of your interests by explaining why you are taking this class was a neat way to go about the assignment. While your professional interests are one part of who you are, I think touching on a few other details about yourself (e.g. where you are from, hobbies, etc.) would have helped me get to know you a little better.

Reply:
I agree with @mwhitfi1 . Also inserting a link or picture here on your past experience or hobbies may be more helpful in showing us who you are, but still nice to know your academic interests!